src/lib/features/events/event-store.ts

```diff
@@ -433,16 +433,16 @@ class EventStore implements IEventStore {
         events.forEach((e) => this.eventEmitter.emit(e.type, e));
     }
 
-    async setCreatedByUserId(batchSize: number): Promise<void> {
+    async setCreatedByUserId(batchSize: number): Promise<number> {
         const API_TOKEN_TABLE = 'api_tokens';
 
         if (!this.flagResolver.isEnabled('createdByUserIdDataMigration')) {
-            return;
+            return -1;
```

Comment:
Find it weird that we return -1 here while we return Promise.resolve() down below... maybe this function should always return the number of rows updated and zero is meaningful, but -1 is not so much.